documentation/functions/record/LOC_BUILDER_STR.md

```diff
@@ -5,12 +5,22 @@ parameters:
   - str
   - alt
   - ttl
+parameter_types:
+  label: string
+  str: string
+  alt: number
+  ttl: number
```

Comment:
Also if possible this should use the `Duration` type (if the TTL is required) or `Duration?` (if it isn’t) to reflect that the helper calls the function to parse values like `"1h"`. (If the helper doesn’t do that it should, but that could be a separate PR)

Reply:
Sure, that can go in.

Reply:
Thanks for the tips - news to me :)